pom.xml

```diff
@@ -79,7 +79,7 @@
         <dependency>
             <groupId>io.swagger</groupId>
             <artifactId>swagger-parser</artifactId>
-            <version>1.0.8-SNAPSHOT</version>
+            <version>1.0.8</version>
```

Comment:
ok - then let's stick with 1.0.8 and make a mental note to look into this "later" :-)